### ISSUE TITLE
Focus

### ISSUE THREAD

**Issue by bcwadsworth** [MCDTeam/Immersion#27]
Right now it seems that we all are not focusing on a main idea that we actually want to focus the mod on. What do we want this mod to do, and what do we need to do to clarify the features we want to include to make this cohesive. For example, when we started we wanted to focus on low-end tech and exploration, but all we are doing right now is high tech or magic stuff that was not the focus originally. Maybe I am wrong, but I think we ought to work on the focus and then make add ons to the focus.

**Comment by realshaunoneill** [comment]
I agree, I think we defiantly do need to get more on track - we need to find a proper means of being able to talk and discuss everything easier instead of this issue board xD  I'm currently moving pcs but I have a bit of work on my laptop I need to push, il do that soon.
